wagtail/contrib/search_promotions/models.py

```diff
@@ -50,22 +76,54 @@ def garbage_collect(cls):
 
     @classmethod
     def get(cls, query_string):
+        """
+        # Gets or creates object matching query_string.
+
+        ## Args:
+            - query_string(str): Query text.
+
+        ## Returns:
+            - First Query object matching query_string.
+
+        ## To do:
+            - Rename get() to get_or_create()
```

Comment:
Same here, please remove the Todo. If you feel it's a critical issue we can add two methods, one with the new name and the current name as an alias. 

Then deprecate the old name in a future release.